src/operation/run_command/mod.rs

```diff
@@ -106,55 +118,3 @@ pub(crate) struct Response {
     cluster_time: Option<ClusterTime>,
     recovery_token: Option<Document>,
 }
```

Comment:
This doesn't seem to be used anywhere now.

Reply:
Good catch, removed. Not sure why clippy didn't pick up on this though.